src/Resources/ServiceRate.php

```diff
@@ -353,4 +359,32 @@ public function getServiceOptions()
     {
         return $this->relationships[self::RELATIONSHIP_SERVICE_OPTIONS]['data'];
     }
+
+    public function setIsDynamic($isDynamic)
+    {
+        $this->attributes[self::ATTRIBUTE_IS_DYNAMIC] = $isDynamic;
+
+        return $this;
+    }
+
+    public function getIsDynamic()
```

Comment:
`isDynamic` reads just a little bit better here :)

```suggestion
    public function isDynamic()
```